merror/domain/status_test.cc

```diff
@@ -132,7 +150,7 @@ TEST(MakeStatus, AcceptStatusOr) {
     passed = true;
     return absl::OkStatus();
   };
-  EXPECT_THAT(F(42), IsOk());
+  EXPECT_TRUE(F(42).ok());
```

Comment:
Maybe define a simple version of `IsOk()` for now? This way it'll be possible to improve the diagnostics in tests by modifying the matcher rather than by tweaking tests.